tests/test_itemized.py

```diff
@@ -54,22 +54,23 @@ def test_sorting(self):
         )
     #This is the only test that the years will have to be bumped when in a new cycle
     #maybe refactor to use some logic based on current year?
-    def test_two_year_transaction_period_default_supplied_automatically(self):
-        receipts = [
-            factories.ScheduleAFactory(
-                report_year=2016,
-                contribution_receipt_date=datetime.date(2016, 1, 1),
-                two_year_transaction_period=2016
-            ),
-            factories.ScheduleAFactory(
-                report_year=2018,
-                contribution_receipt_date=datetime.date(2018, 1, 1),
-                two_year_transaction_period=2018
-            ),
-        ]
-
-        response = self._response(api.url_for(ScheduleAView))
-        self.assertEqual(len(response['results']), 1)
+    # remove this one because we are removing 2-year period restriction
```

Comment:
Please remove this test rather than commenting it out.

Also, can you please add an automated test for multiple two year periods?

Reply:
added multiple two year periods tests for both sched_a and sched_b. Thx.